### PR TITLE
Create scroll-to-top.md

### LINKED ISSUE #1448: Add documentation for Scroll to Top feature
Feature ticket in Spartacus repo ->  [link](https://github.com/SAP/spartacus/issues/15356)

### PR DESCRIPTION
**Pull request by BenjaminEmiliani**
Closes #1448

### CODE REVIEW

File: _pages/dev/features/scroll-to-top.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+**Table of Contents**
+
+- This will become a table of contents (this text will be scrapped).
+{:toc}
+
+---
+
+## Enabling Scroll to Top
+
+Scroll to top is part of the `@spartacus/storefront` library, and installed during the initial setup. To enable this feature in your storefront add the scroll to top CMS data to a content slot using ImpEx.
```
Comment (plabadie) [inline _pages/dev/features/scroll-to-top.md:28]: Suggestion:

To enable this feature in your storefront, add the Scroll to Top CMS Component to a content slot in the web site pages you wish to use the feature with.  To add Scroll to Top to the whole web site, you can add the Scroll to Top component in a shared content slot, like the footer.

File: _pages/dev/features/scroll-to-top.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+Scroll to top is part of the `@spartacus/storefront` library, and installed during the initial setup. To enable this feature in your storefront add the scroll to top CMS data to a content slot using ImpEx. 
+
+### CMS Component
+
+Scroll to top is CMS-driven and consists of the following CMS component:
+
+- `ScrollToTopComponent`
+
+If you are using the version 5.0 or greater of [{% assign linkedpage = site.pages | where: "name", "spartacussampledata-extension.md" %}{{ linkedpage[0].title }}]({{ site.baseurl }}{% link _pages/install/spartacussampledata-extension.md %}), the scroll to top component is already enabled. However, if you decide not to use the `spartacussampledata` extension, you can enable scroll to top through ImpEx.
```
Comment (plabadie) [inline _pages/dev/features/scroll-to-top.md:36]: if you decide not to use the `spartacussampledata` extension, you can create the Scroll to Top CMS component with ImpEx.

File: _pages/dev/features/scroll-to-top.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+```text
+$contentCatalog=powertools-spaContentCatalog
+$contentCV=catalogVersion(CatalogVersion.catalog(Catalog.id[default=$contentCatalog]),CatalogVersion.version[default=Staged])[default=$contentCatalog:Staged]
+```
+
+#### Adding Scroll to Top CMS Component Manually
+
+This section describes how to add the scroll to top CMS component to Spartacus using ImpEx.
+
+You can enable the **Scroll to Top** component by adding the CMS data with the following ImpEx:
```
Comment (plabadie) [inline _pages/dev/features/scroll-to-top.md:49]: Suggestion:

You can create the **Scroll to Top** component with the following ImpEx:

File: _pages/dev/features/scroll-to-top.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+```text
+INSERT_UPDATE CMSFlexComponent;$contentCV[unique=true];uid[unique=true];name;flexType;&componentRef
+;;ScrollToTopComponent;Scroll To Top Component;ScrollToTopComponent;ScrollToTopComponent
+```
+
+You can add the **Scroll to Top** component by inserting the CMS component in the **Footer** slot with the following ImpEx:
```
Comment (plabadie) [inline _pages/dev/features/scroll-to-top.md:56]: You can enable the **Scroll to Top** feature by adding the **Scroll to Top** component

File: _pages/dev/features/scroll-to-top.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+    },
+  },
+}),
+```
+
+In the above example, `ScrollBehavior.AUTO`, is part of an `enum` located in `@spartacus/core`.
+
+### Display Threshold
+
+Display threshold determines at which scroll position (0 at the top and increases as user scrolls down) will the button be displayed on the screen. By default, the display threshold is set to half the height of the window, but you can modify this value to by providing a different configuration. You can provide your configuration as shown in the following example:
```
Comment (plabadie) [inline _pages/dev/features/scroll-to-top.md:94]: Suggestion:

Display threshold determines after how many pixels from the top of the page will the button be displayed on the screen. By default, the display threshold is set to half the height of the window, but you can modify this value to by providing an absolute number via configuration. You can provide your configuration as shown in the following example:

File: _pages/dev/features/scroll-to-top.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+        displayThreshold: 1000,
+      }
+    },
+  },
+}),
+```
+
+In the above example, display threshold is fixed and will display the scroll to top button at a `window.scrollY` value of 1000.
+
+Both of these examples can be combined into a single configuration.
```
Comment (plabadie) [inline _pages/dev/features/scroll-to-top.md:110]: Suggestion:

Show an example.

File: _pages/dev/features/scroll-to-top.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
+---
+title: Scroll to Top
+feature:
+  - name: Scroll to Top
+    spa_version: 5.0
+    cx_version: 2105
```
Comment (plabadie) [inline _pages/dev/features/scroll-to-top.md:6]: Doesn't require a minimum CX version I believe.  It's frontend only feature.

**Review by gladius-mtl — APPROVED** [review]

**Pull request merged**